websites/F/Facebook/metadata.json

```diff
@@ -21,40 +21,68 @@
 		"zh_CN": "与朋友、家人和其他你认识的人联系。分享照片和视频、发送消息和获取更新。",
 		"vi_VN": "Kết nối với bạn bè, người thân và những người khác bạn biết. Chia sẻ hình ảnh và video, gửi tin nhắn và nhận các thông báo."
 	},
-	"url": "www.facebook.com",
-	"version": "3.6.4",
+	"url": ["www.facebook.com", "web.facebook.com"],
+	"version": "3.7.0",
 	"logo": "https://i.imgur.com/nS9sZn6.png",
 	"thumbnail": "https://i.imgur.com/iftQq6r.jpg",
 	"color": "#4267B2",
 	"category": "socials",
-	"tags": [
-		"friends",
-		"family"
-	],
+	"tags": ["friends", "family"],
```

Comment:
This file is not formatted properly, reformat with `yarn format`

Reply:
> This file is not formatted properly, reformat with `yarn format`

i know